utils/format.go

```diff
@@ -859,87 +843,78 @@ func FormatTimestampTs(ts time.Time) template.HTML {
 	} else if status == "exiting_online" {
-		return "<b>Exiting</b> <i class=\"fas fa-power-off fa-sm text-success\"></i>"
+		return `Exiting`
 	} else if status == "exiting_offline" {
-		return "<span data-toggle=\"tooltip\" title=\"No attestation in the last 2 epochs\"><b>Exiting</b> <i class=\"fas fa-power-off fa-sm text-danger\"></i></span>"
+		return `<span class="text-danger" data-toggle="tooltip" title="No attestation in the last 2 epochs">Exiting</span>`
 	} else if status == "slashing_online" {
-		return "<b>Slashing</b> <i class=\"fas fa-power-off fa-sm text-success\"></i>"
+		return `<span class="text-danger">Slashing</span>`
 	} else if status == "slashing_offline" {
-		return "<span data-toggle=\"tooltip\" title=\"No attestation in the last 2 epochs\"><b>Slashing</b> <i class=\"fas fa-power-off fa-sm text-danger\"></i></span>"
+		return `<span class="text-danger" data-toggle="tooltip" title="No attestation in the last 2 epochs">Slashing</span>`
 	} else if status == "exited" {
-		return "<span><b>Exited</b></span>"
+		return "Exited"
 	} else if status == "slashed" {
-		return "<span><b>Slashed</b></span>"
-	}
-	return "<b>Unknown</b>"
-}
-
-func formatPool(tag []string) string {
-	if len(tag) > 1 {
-		tagType := tag[0]
-		tagName := strings.Split(tag[len(tag)-1], " ")
-		if len(tagName) > 1 {
-			_, err := strconv.ParseInt(tagName[len(tagName)-1], 10, 64)
-			if err == nil {
-				name := ""
-				for _, s := range tagName[:len(tagName)-1] {
-					if s == "-" {
-						continue
-					}
-					name += s + " "
-				}
-				return fmt.Sprintf(`<a href='/pools' style="all: unset; cursor: pointer;" data-toggle="tooltip" title="This validator is part of a staking-pool"><span style="font-size: 18px;" class="bg-light text-dark badge-pill pr-2 pl-0 mr-1"><span class="bg-dark text-light rounded-left mr-1 px-1">%s</span> %s</span></a>`, tagType, name)
-			}
-		}
-		return fmt.Sprintf(`<a href='/pools' style="all: unset; cursor: pointer;" data-toggle="tooltip" title="This validator is part of a staking-pool"><span style="font-size: 18px;" class="bg-light text-dark badge-pill pr-2 pl-0 mr-1"><span class="bg-dark text-light rounded-left mr-1 px-1">%s</span> %s</span></a>`, tagType, tag[len(tag)-1])
-	}
-	return ""
-}
-
-func formatSpecialTag(tag string) string {
-	special_tag := strings.Split(tag, ":")
-	if len(special_tag) > 1 {
-		if special_tag[0] == "pool" {
-			return formatPool(special_tag)
-		}
+		return `<span class="text-danger">Slashed</span>`
 	}
-	return fmt.Sprintf(`<span style="font-size: 18px;" class="badge bg-dark text-light mr-1">%s</span>`, tag)
+	return "Unknown"
 }
 
 // FormatValidatorTag will return html formated text of a validator-tag.
 // Depending on the tag it will describe the tag in a tooltip and link to more information regarding the tag.
-func FormatValidatorTag(tag string) template.HTML {
+func formatValidatorTag(tag string) string {
```

Comment:
Why change the return type here?
I would rather keep it `template.HTML` here and change the `str` below where it is used also to a `template.HTML`

Reply:
There is no reason to keep it as template.HTML since that function is not available as template function anymore (it never used as that). In the old code the return value of this function was cast to string anyways so I changed the return type. Won't change unless you insist.